e2e/README.md

```diff
@@ -46,7 +46,7 @@ following environment variables can be exported to customize kubernetes deployme
 | ENV                | Description                                      | Default                                                            |
 | ------------------ | ------------------------------------------------ | ------------------------------------------------------------------ |
 | MINIKUBE_VERSION   | minikube version to install                      | latest                                                             |
-| KUBE_VERSION       | kubernetes version to install                    | v1.13.0                                                            |
+| KUBE_VERSION       | kubernetes version to install                    | v1.14.0                                                            |
```

Comment:
`scripts/minikube.sh` actually uses v1.14.2 already by default.

Reply:
Updated and bumped the `scripts/minikube.sh` default to 1.14.10 to be in sync with TravisCI